docs/README.rst

```diff
@@ -0,0 +1,98 @@
+.. _readme:
+
+exim-formula
+============
+
+|img_travis| |img_sr|
+
+.. |img_travis| image:: https://travis-ci.com/saltstack-formulas/exim-formula.svg?branch=master
+   :alt: Travis CI Build Status
+   :scale: 100%
+   :target: https://travis-ci.com/saltstack-formulas/exim-formula
+.. |img_sr| image:: https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg
+   :alt: Semantic Release
+   :scale: 100%
+   :target: https://github.com/semantic-release/semantic-release
+
+``exim`` scans log files for malicious activity and executes actions based on what it finds.
```

Comment:
@n-rodriguez So you're using `fail2ban-formula` as your "template", huh?!

Reply:
Ahah! you caught me!